webapp/src/component/layout/Notifications/NotificationsPopup.tsx

```diff
@@ -167,11 +167,12 @@ export const NotificationsPopup: React.FC<NotificationsPopupProps> = ({
             <T keyName="notifications-empty" />
           </ListItem>
         )}
-        {(notificationsLoadable.isFetching ||
-          notificationsLoadable.hasNextPage) && (
-          <ListItem>
-            <BoxLoading width="100%" />
-          </ListItem>
+        {[1, 2, 3, 4, 5].map(
```

Comment:
This is not very readable. I guess it's there because of the skeletons, right? 
Move it to separate component named `NotificationSkeletons` or something.

Then move it to separate component and don't mix the skeleton with the real notification item. 

This check for fetching, should happen before the `.map` function. Rendering 5x nothing makes no sense.

Reply:
I am adding the skeletons to the end of the list in case there are some more notifications left to be loaded.

When a user scrolls to them, it loads more.

![image](https://github.com/user-attachments/assets/6cdde057-37c7-4255-bb5d-5cb73be99b30)

I updated the condition in the front of the map function, thanks for the catch.